lib/service/Service.ts

```diff
@@ -181,6 +182,36 @@ class Service {
     return balances;
   }
 
+  /** Gets the trading limits (max outbound and inbound capacities for a distinct channel) for a given currency. */
+  public tradingLimits = async (args: { currency: string }) => {
+    const { currency } = args;
+    const tradingLimits = new Map<string, TradingLimits>();
+
+    if (currency) {
+      argChecks.VALID_CURRENCY(args);
+
+      const swapClient = this.swapClientManager.get(currency.toUpperCase());
+      if (swapClient) {
+        const tl = await swapClient.tradingLimits(currency);
```

Comment:
I'd prefer `tradingLimit` to `tl` if that's alright with you, just a bit clearer that way. Same for where it's used a few lines below.